src/stylesheets/core/mixins/_add-checkbox-placeholder.scss

```diff
@@ -1,14 +1,15 @@
 @mixin add-checkbox-placeholder {
-  $placeholder-width: units(3);
-  $placeholder-margin: units(0.5);
+  $placeholder-width: units(4);
+  $placeholder-margin: units(1);
 
   text-indent: ($placeholder-width + $placeholder-margin) * -1;
 
   &::before {
     content: ' ';
     display: inline-block;
-    height: units(1.5);
-    margin-right: $placeholder-margin;
+    height: units(2);
+    margin-left: units(neg-05);
```

Comment:
You can use `units(-0.5)` here. (If it doesn't work, that's a problem.)

Reply:
Done in 4487300.